.github/workflows/ci.yml

```diff
@@ -195,13 +184,13 @@ jobs:
           version: ${{ env.MISE_VERSION }}
       - name: Install Tools
         run: |
-          ./scripts/install-rust.sh
-          ./scripts/install-go.sh
-          ./scripts/install-protoc.sh
-          ./scripts/install-node.sh
+          go run ./cmd/dev install rust go protoc node
```

Comment:
Wondering if we actually need node here.

Reply:
We do, unfortunately, as we are running `npm run proto-gen` as part of the build.